Libraries/NativeAnimation/RCTNativeAnimatedModule.m

```diff
@@ -144,7 +154,7 @@ - (dispatch_queue_t)methodQueue
   [_activeAnimations addObject:animationDriver];
   _animationDrivers[animationId] = animationDriver;
   [animationDriver startAnimation];
-  [self startAnimation];
+  [self startAnimationLoopIfNeeded];
```

Comment:
👍 Updated with a slightly more optimized version that doesn't call updateAnimation and does the minimal amount of work needed to update the views since we don't need to do everything updateAnimation does in this case.